src/openzaak/components/autorisaties/forms.py

```diff
@@ -322,12 +322,13 @@ def _validate_external_types(self, component):
                 )
                 self.add_error("externe_typen", error)
 
-    def get_types(self):
-        component = self.cleaned_data["component"]
+    def get_types(self, component):
```

Comment:
I don't quite understand why to change 'get_types' parameters?

Reply:
well, all your other helpers had that argument and everytime `get_types` was called, the component was already known (via the same line of code that I removed here).